test_pkm.py

```
@@ -110,59 +114,57 @@ async def test_processor(config, storage):
     removed = scheduler.remove_job("test_job")
     console.print(f"✅ Test job removed: {removed}")
 
 
 async def main():
     """Main test function."""
-    console.print(Panel.fit(
-        "[bold cyan]🧪 PKM Module Test Suite[/bold cyan]",
-        border_style="cyan"
-    ))
-    
+    console.print(Panel.fit("[bold cyan]🧪 PKM Module Test Suite[/bold cyan]", border_style="cyan"))
```

Ruff (0.14.10)

Undefined name `Panel`

(F821)